packages/testing/src/chai-retry-plugin/helpers.ts

```diff
@@ -1,9 +1,22 @@
 import Chai from 'chai';
-import { timeout as timeoutPromise, sleep } from 'promise-assist';
+import { timeout as timeoutPromise, sleep as sleepPromise } from 'promise-assist';
+import { adjustTestTime, mochaCtx } from '../mocha-ctx';
 
 import { chaiMethodsThatHandleFunction } from './constants';
 import type { RetryAndAssertArguments } from './types';
 
+export const addTimeoutSafetyMargin = (ms: number) => mochaCtx() && adjustTestTime(ms);
```

Comment:
It's "half-clear" what this is for... at least for me.
what do you think about leaving a short comment which explains that?

Reply:
@yurii-ve fixed, didn't add it cause imported `mochaCtx` and `adjustTestTime` already had comments with explanations available with InteliSense